include/wx/setup_inc.h

```diff
@@ -919,6 +919,15 @@
 // the native control limitations.
 #define wxUSE_NATIVE_DATAVIEWCTRL 1
 
+// Use generic version of wxAnimationCtrl and supporting classes, even if a 
+// native one is available?
```

Comment:
This seems reversed. I.e. the description corresponds to the value of 0 of this option or something like that.